src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -1934,19 +1950,47 @@ def _add_metrics_update_callback(self, phase: Phase):
         """
         self.phase_callbacks.append(MetricsUpdateCallback(phase))
 
-    def _initialize_neptune_run(self, additional_configs_to_log: Dict = None):
+    def _initialize_neptune_run_test(self, configs: Dict = None):
```

Comment:
```suggestion
    def _initialize_neptune_run_eval(self, configs: Dict = None):
```
test is confusing because it can indicate test split and You evaluate on val

Reply:
that is true - I can change it. However, the Trainer._initialize_neptune_run_train() function is called in Trainer.train(), and Trainer._initialize_neptune_run_test() is called in Trainer.test() - so it is consistent now. pls let me know what do you think

Reply:
ok ;)